src/trackers.js

```diff
@@ -9,9 +9,9 @@ class Trackers {
     }
 
     processTrackerList (data) {
-        for (let name in data) {
```

Comment:
all these years and it's still not obvious to me why const works here. I guess we are not changing value but destroying/creating new variable with every loop?

Reply:
Yeah exactly that.

We're stating that _name_ is non mutable but the for expression runs in isolation so we just redefine a new variable with called _name_ and give it a value.

Reply:
I used to be kinda pro relaxing rules for ESLint here due to it being a little confusing syntactically. But unless we need mutability of the variable I'm happier now preventing accidental side effects.